test_scripts/Smoke/Resumption/ATF_Resumption_FULL_IGNITION_OFF.lua

```diff
@@ -0,0 +1,129 @@
+
+--  1. Used preconditions
+--  App is registered and activated on HMI
+
+--  2. Performed steps
+--  Perform iginition off
+--  Perform ignition on
+--
+--  Expected behavior:
+--  1. SDL save all App's persistent data to app_info.dat file and sends to HMI OnSDLClose
+--  2. App is registered, SDL sends OnAppRegistered with the same HMI appID as in last ignition cycle, 
+--     then sends BasicCommunication.ActivateApp to HMI and after success response from HMI, SDL sends to App OnHMIStatus(FULL)
+
+-- [[ Required Shared Libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonStepsResumption = require('test_scripts/Smoke/Resumption/commonStepsResumption')
+local mobile_session = require('mobile_session')
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/dummy_connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+-- [[Local variables]]
+local default_app_params = config.application1.registerAppInterfaceParams
+local default_app = nil -- will be initialized after application registration
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+
+function Test:Start_SDL_With_One_Activated_App()
+  self:runSDL()
+  commonFunctions:waitForSDLStart(self):Do(function()
+    self:initHMI():Do(function()
+      commonFunctions:userPrint(35, "HMI initialized")
+      self:initHMI_onReady():Do(function ()
+        commonFunctions:userPrint(35, "HMI is ready")
+        self:connectMobile():Do(function ()
+          commonFunctions:userPrint(35, "Mobile Connected")
+          self:startSession():Do(function ()
+            commonFunctions:userPrint(35, "App is registered")
+            default_app = self.applications[default_app_params.appName]
+            commonSteps:ActivateAppInSpecificLevel(self, default_app)
+            EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL",audioStreamingState = "AUDIBLE", systemContext = "MAIN"})
+            commonFunctions:userPrint(35, "App is activated")
+          end)
+        end)
+      end)
+    end)
+  end)
+end
+
+function Test:AddCommand()
+  local cid = self.mobileSession:SendRPC("AddCommand", { cmdID = 1, vrCommands = {"OnlyVRCommand"}})
+  local on_hmi_call = EXPECT_HMICALL("VR.AddCommand", {cmdID = 1, type = "Command",
+                      vrCommands = {"OnlyVRCommand"}})
+  on_hmi_call:Do(function(_, data)
+    self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  EXPECT_RESPONSE(cid, { success = true, resultCode = "SUCCESS" })
+  EXPECT_NOTIFICATION("OnHashChange"):Do(function(_, data)
+    self.currentHashID = data.payload.hashID
+  end)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Transport disconnect <30s before IGNITION_OFF. Media app resume at FULL level")
+
+function Test:Close_Session()
```

Comment:
@istoimenova I update script according to [HMILevel Resumption]: Conditions to resume app to FULL in the next ignition cycle. And I think that we don't need time verification if time **less** then 30, it makes sense for example in case when time **more** than 30.